modules/tinymce/src/plugins/table/test/ts/browser/ExcessColsTest.ts

```diff
@@ -0,0 +1,93 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.plugins.table.ExcessColsTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ Plugin, Theme ], true);
+
+  const assertBasicTablePresence = (editor: Editor, tdCount: number) => {
+    TinyAssertions.assertContentPresence(editor, {
+      td: tdCount,
+    });
+  };
+
+  // Note: THe goal of these tests is to make sure an exception is not thrown and each table operation works
+  // when there are more col elements than actual columns
+  Arr.each([
+    {
+      label: 'more cols',
+      tableHTML: '<table><colgroup><col /><col /><col /><col /></colgroup><tbody><tr><td>1</td><td>2</td></tr><tr><td>3</td><td>4</td></tr></tbody></table>',
```

Comment:
Can we please rename this to `tableHtml` as that's more consistent with our regular variable naming conventions?

Reply:
Done